lib/Stripe/BitcoinTransaction.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+class Stripe_BitcoinTransaction extends Stripe_ApiResource
```

Comment:
They do seem to be listable and fetchable, so do we want to have `retrieve` and `all` methods?